src/components/reservation/ReservationFormListItem.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { useText } from "../../core/utils/text";
+
+interface ReservationFormListItemProps {
+  icon: string;
+  title: string;
+  text: string;
+}
+
+const ReservationFormListItem: React.FC<ReservationFormListItemProps> = ({
+  icon,
+  title,
+  text
+}) => {
+  const t = useText();
+  return (
+    <div className="reservation-modal-list-item">
+      <img src={icon} alt="" />
+      <div className="reservation-modal-list-item-text">
+        <h3 className="text-header-h5">{title}</h3>
+        <p className="text-small-caption">{text ?? `Mangler data`}</p>
+      </div>
+      {/* <button
+        type="button"
+        className="link-tag text-small-caption cursor-pointer"
+      >
+        {t("shiftText")}
+      </button> */}
```

Comment:
Please go though this code again.

- `t` is declared but never used
- There should not be commented out code in the codebase unless there is a reason to. If you think the code is should then leave it be with a `TODO` comment for the click handler or the like.
- "Mangler data" is not translated. I am not sure how it can be false anyway. Perhaps if the string is empty? Are we sure we want to print "Mangler data" here anyway?